Hearthstone Deck Tracker/Hearthstone/Watchers.cs

```diff
@@ -52,4 +55,9 @@ public class HearthMirrorArenaProvider : IArenaProvider
 		public ArenaInfo GetArenaInfo() => DeckImporter.FromArena(false);
 		public HearthMirror.Objects.Card[] GetDraftChoices() => Reflection.GetArenaDraftChoices()?.ToArray();
 	}
+
+	public class HearthMirrorFriendlyChallengeDialogVisibilityProvider : IDialogVisibilityProvider
```

Comment:
We can probably shorten this to at least `HearthMirrorFriendlyChallengeProvider ` too.